lib/ts/recipe/dashboard/api/userdetails/userSessionsGet.ts

```diff
@@ -30,7 +31,7 @@ export const userSessionsGet: APIFunction = async (
         });
     }
 
-    const response = await Session.getAllSessionHandlesForUser(userId, userContext);
+    const response = await Session.getAllSessionHandlesForUser(userId, undefined, userContext);
```

Comment:
search for getAllSessionHandlesForUser in the SDK and make sure that we are passing 3 args everywhere

Reply:
Done, didn't find any other